lib/initInstance.js

```diff
@@ -140,6 +161,7 @@ if (siteStorage.length == 0 && instanceStorage.length == 0) {
   } else {
     execSync("cp -r "+path.join(config.productDir, 'ZLUX', 'pluginStorage')+" "+path.join(config.instanceDir, 'ZLUX'));
     execSync("chmod -R 770 "+instancePluginStorage);
+    generateComponentJson()
```

Comment:
there is another process.exit(0) above for windows. You might need it there too?